app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from typing import Union
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Union[str, None]:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All visitors should be vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine expired")
+
+        if visitor["wearing_a_mask"] is False:
```

Comment:
Use `not visitor["wearing_a_mask"]` instaed of `visitor["wearing_a_mask"] is False` it is better approach